Source/WebCore/html/track/TextTrack.h

```diff
@@ -109,7 +109,11 @@ class TextTrack : public TrackBase, public EventTarget, public ActiveDOMObject {
     virtual bool isEasyToRead() const { return false; }
 
     int trackIndex();
-    void invalidateTrackIndex();
+    inline void invalidateTrackIndex()
```

Comment:
You don't need the `inline` keyword in the method implementation is already inlined, like it is here.